source/conf.py

```diff
@@ -88,9 +88,10 @@ def make_ref(ref_str, ref_num, ref_sufix):
     make_ref("SYCL_SPEC_HEADER_FILES", "4.3", "#sec:headers-and-namespaces")
     + make_ref("SYCL_SPEC_DEVICE_SELECTORS", "4.6.1.1", "#sec:device-selector")
     + make_ref("SYCL_SPEC_PLATFORM", "4.6.2", "#sec:platform-class")
+    + make_ref("SYCL_SPEC_DEVICE", "4.6.4", "#sec:device-class")
+    + make_ref("SYCL_SPEC_MEMORY_ORDERING", "3.8.3.1", "#_memory_ordering")
+    + make_ref("SYCL_SPEC_MEMORY_SCOPE", "3.8.3.2", "#_memory_scope")
```

Comment:
I put one more change in make_ref...in order that we can create any view string,not only static Section, I have added :
def make_ref(ref_str, ref_view, ref_sufix):
    return (
        f".. |{ref_str}| replace:: SYCL Specification "
        f"`{ref_view} <{sycl_ref_url}{ref_sufix}>`__"
        f"\n"
    )
In call you should explicit put full view :
    make_ref(
        "SYCL_SPEC_HEADER_FILES", "**Section 4.3**", "#sec:headers-and-namespaces"
    )
or
    + make_ref(
        "SYCL_SPEC_QUEUE_PROPERTIES", "**Table 31**", "#table.properties.queue"
    )